public/app/core/components/NavBar/NavBar.tsx

```diff
@@ -64,7 +64,7 @@ export const NavBar = React.memo(() => {
     {
       id: 'home',
       text: 'Home',
-      url: config.appSubUrl || '/',
+      url: config.bootData.user.isSignedIn ? config.appSubUrl || '/' : '/login',
```

Comment:
do our routes automatically handle adding the base url if served a subpath, or would we need to add that here?

Reply:
good point

Reply:
seems like they do 👍